Nio/Authentication/LoginView.swift

```diff
@@ -144,7 +168,7 @@ private struct FormTextField: View {
                     .padding()
                     .textContentType(.password)
             } else {
-                TextField(title, text: $text)
+                TextField(title, text: $text, onEditingChanged: onEditingChanged ?? { (editingChanged) in })
```

Comment:
Also dropping the unused argument here.

```suggestion
                TextField(title, text: $text, onEditingChanged: onEditingChanged ?? { _ in })
```